desktop-exporter/app/routes/trace-view.jsx

```diff
@@ -40,22 +44,29 @@ function WaterfallView(props) {
     );
 }
 
-function DetailView() {
+function DetailView(props) {
```

Comment:
I think I would call the prop `span` instead of `spanData` for concision. 

Here is a potentially more idiomatic way of writing this:

```jsx
function DetailView(props) {
    let {span} = props;
    return (
        <div className='detail'>
            <pre>
                Name: {span.name}
                Kind: {span.kind}
                Start: {span.startTime}
                End: {span.endTime}
            </pre>
        </div>
    );
}
```